### PR TITLE
Translation of documentation

### PR DESCRIPTION
**Pull request by Alek5andr-Kotov**
### Changelog entry 

Documentation on reading from topics in a transaction.

### Changelog category 

* Documentation (changelog entry is not required)
* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/docs/en/core/reference/ydb-sdk/topic.md [hunk 1]
```diff
@@ -1160,6 +1160,8 @@ You can read messages without a [commit](#no-commit) as well. In this case, all
 
 Information about which messages have already been processed can be [saved on the client side](#client-commit) by sending the starting consumer offset to the server when creating a new connection. This does not change the consumer offset on the server.
 
+You can use [transactions](#read-tx). In this case, the reading offset will change only when the transaction is commited. When you connect again, all uncommited messages will be read.
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/topic.md:1163]:
```suggestion
Data from topics can be read in the context of [transactions](#read-tx). In this case, the reading offset will only advance when the transaction is committed. On reconnect, all uncommitted messages will be read again.
```

File: ydb/docs/en/core/reference/ydb-sdk/topic.md [hunk 2]
```diff
@@ -1565,6 +1567,60 @@ Reading progress is usually saved on a server for each Consumer. However, such p
 
 {% list tabs %}
 
+- C++
+
+  Before reading messages, the client code must pass a reference to the transaction object to the settings of the reading session.
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/topic.md:1572]:
```suggestion
  Before reading messages, the client code must pass a transaction object reference to the reading session settings.
```

File: ydb/docs/en/core/reference/ydb-sdk/topic.md [hunk 3]
```diff
@@ -1565,6 +1567,60 @@ Reading progress is usually saved on a server for each Consumer. However, such p
...
+          // process the event and write results to a table
+      }
+
+      NYdb::NTable::TCommitTxSettings commitSettings;
+      auto commitResult = Transaction.Commit(commitSettings).GetValueSync();
+  ```
+
+{% note warning %}
+
+  When processing `events`, you do not need to explicitly confirm processing for events of the `TDataReceivedEvent`.
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/topic.md:1597]:
```suggestion
  When processing `events`, you do not need to confirm processing for `TDataReceivedEvent` events explicitly.
```

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/reference/ydb-sdk/topic.md (hunk 1 above)
Comment (Alek5andr-Kotov) [reply ydb/docs/en/core/reference/ydb-sdk/topic.md]: fixed
https://github.com/ydb-platform/ydb/pull/4918/commits/2884b64e8c40cd192b9ea61db595b75a08017044

File: ydb/docs/en/core/reference/ydb-sdk/topic.md (hunk 2 above)
Comment (Alek5andr-Kotov) [reply ydb/docs/en/core/reference/ydb-sdk/topic.md]: fixed
https://github.com/ydb-platform/ydb/pull/4918/commits/2884b64e8c40cd192b9ea61db595b75a08017044

File: ydb/docs/en/core/reference/ydb-sdk/topic.md (hunk 3 above)
Comment (Alek5andr-Kotov) [reply ydb/docs/en/core/reference/ydb-sdk/topic.md]: fixed
https://github.com/ydb-platform/ydb/pull/4918/commits/2884b64e8c40cd192b9ea61db595b75a08017044

**Review by fomichev3000 — DISMISSED** [review]

**Review by blinkov — APPROVED** [review]

**Pull request merged**